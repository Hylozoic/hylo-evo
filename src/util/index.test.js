import { formatDatePair } from './index'
import moment from 'moment'

describe('formatDatePair', () => {
  it('displays differences of dates', () => {
    const d1 = moment(1551908483315).month(1).day(1).hour(18)
    const d2 = moment(d1).hour(21)
    const d3 = moment(d2).day(2)
    const d4 = moment(d3).month(2)

    expect(formatDatePair(d1)).toMatchSnapshot()
    expect(formatDatePair(d1, d2)).toMatchSnapshot()
    expect(formatDatePair(d1, d3)).toMatchSnapshot()
    expect(formatDatePair(d1, d4)).toMatchSnapshot()
  })
})