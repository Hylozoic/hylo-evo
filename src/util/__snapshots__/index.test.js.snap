// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatDatePair displays differences of dates 1`] = `"Mon, Feb 4 at 6:41PM"`;

exports[`formatDatePair displays differences of dates 2`] = `"Mon, Feb 4 at 6:41PM - 9:41PM"`;

exports[`formatDatePair displays differences of dates 3`] = `"Mon, Feb 4 at 6:41PM - Tue 5 at 9:41PM"`;

exports[`formatDatePair displays differences of dates 4`] = `"Mon, Feb 4 at 6:41PM - Tue, Mar 5 at 9:41PM"`;
